x11-misc/libinput-gestures/libinput-gestures-9999.ebuild

```diff
@@ -36,3 +50,8 @@ pkg_postinst() {
 		elog "${PN}-setup script supports Plasma 5 via kde-plasma/kde-cli-tools:5."
 	fi
 }
+
+pkg_postrm() {
+	gnome2_icon_cache_update
+	xdg_desktop_database_update
```

Comment:
There is no "MimeType=" in libinput-gestures.desktop.
Looks like xdg_desktop_database_update shouldn't be here.
https://wiki.gentoo.org/wiki/Notes_on_ebuilds_with_GUI